manifests/charts/istio-control/istio-discovery/templates/remote-istiod-endpoints.yaml

```diff
@@ -1,12 +1,13 @@
+# This file is only used for remote `istiod` installs.
```

Comment:
Does someone ever install `istio-discovery` in a remote scenario? Or does this only need to be in istio-remote

Reply:
I don't believe so. We don't in our tests, but I am trying not to make too many assumptions in a single PR.

We have other resources in `istio-discovery` that also reference/are conditional on `remotePilotAddress`, and are copied to `istio-remote` - those also probably should just go in `istio-remote`, and do not need to be copied from `istio-discovery` at all (and I suspect if you installed the current `istiod` with remotePilotAddress set, the chart would simply break).

Did not do that here, as detangling `istio-discovery` and `istio-remote` is probably a larger concern, just getting stuff out of `base` that doesn't need to be there.

Reply:
See also https://github.com/istio/istio/pull/53016#discussion_r1775510055